test/test.js

```diff
@@ -9,41 +9,41 @@ import presetLintNode from "../index.js";
 import { read } from "to-vfile";
 import { reporter } from "vfile-reporter";
 
+import fixturesUrl from "./fixtures/fixturesUrl.js";
+
 const linter = unified()
   .use(remarkParse)
   .use(presetLintNode)
   .use(remarkStringify);
 
 const handleError = (err) => {
   console.error(err);
-  process.exit(1);
+  process.exitCode = 1;
 };
 
 // Top-level await is not supported in Node.js 12.x. Once we no longer support
 // 12.x, this test file can be improved with top-level await.
 
 // Test that correctly-formatted markdown is ok.
 (async () => {
-  const file = await read(new URL("./fixtures/ok.md", import.meta.url));
+  const file = await read(new URL("./ok.md", fixturesUrl));
```

Comment:
Why is the fixtures being dropped from the path here? Is it related to the move of that import?

Reply:
We're importing the `fixturesUrl`, which is a file: URL leading to the `fixtures/` folder, so yes it is related to the move of that import :) (`import.meta.url` is an absolute URL leading to the module where it's executed, not relative to the cwd or the module importing it).

Reply:
OK, maybe that is just more confusing for me 😄 
Would pulling it back up to the test root, and just ignoreing the file path in eslintignore (instead of the directory) make sense? Don't feel strongly about it, but it wouldn't be obvious on why the paths worked like this if I was going to write a new test

Reply:
Would it help if we renamed `fixturesUrl` into something else? Something that makes it obvious that the relative URL is based on the `fixtures` directory.